src/oss-find-squats-lib/Mutators/RemoveSeparatedSectionMutator.cs

```diff
@@ -9,13 +9,13 @@ namespace Microsoft.CST.OpenSource.FindSquats.Mutators
     /// <summary>
     /// Generates mutations for if a character was added after a separator was used.
```

Comment:
Is this comment still correct? It seems the purpose of this mutator was changed.

Reply:
Thanks for the catch! Fixed.